examples/options/include/traccc/options/generation.hpp

```diff
@@ -38,8 +42,8 @@ class generation : public interface {
     opts::value_array<float, 2> phi_range{-180.f, 180.f};
     /// Range of eta
     opts::value_array<float, 2> eta_range{-2.f, 2.f};
-    /// Charge of particles
-    float charge{-1.f};
+    /// PDG number for particle type (Default: muon)
+    unsigned int pdg_number = 13;
```

Comment:
Why would this be `unsigned int`? The PDG number is signed no?

Reply:
Good catch. will fix

Reply:
Fixed to int